.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,60 @@
+## Summary of Changes
+
+Please write a sentence or two summarizing the proposed change.
+
+Addresses [ORCA-XX: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/ORCA-XXX)
+
+## Changes
+
+* Detailed list or prose of changes
+* ...
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Test A
+- [ ] Test B
+
+**Test Configuration**:
+* Firmware version:
+* Hardware:
+* Toolchain:
+* SDK:
+
+## Checklist:
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] Any dependent changes have been merged and published in downstream modules
+
```

Comment:
Possibly add a bullet that specifies all security checks have passed.
```
- [] My code has passed security scanning
```
We could expand on this in the developer docs to include information on stuff like Snyk or some python specific tools like black, safetyd, pylint, bandit, git-secrets, or others. Depending on what security best practices we would like to ask for that can easily be added into the pipeline for testing.

Reply:
I agree. I don't think we have a suite of standard tools selected, but we can add those as we adopt them, if that makes sense to you.